### PR TITLE
Automatic port of Microsoft.Extension.Caching.Distributed docs

### PR DESCRIPTION
**Pull request by carlossanlop**
Area owners of Microsoft.Extensions.Caching: @maryamariyan @eerhardt

### CODE REVIEW

**Comment by eerhardt** [comment]
Tagging @Tratcher - who is the owner if this project according to https://github.com/dotnet/extensions/blob/1991801bbd2a6bcc35ddb0993ca29b0d8053f571/.github/CODEOWNERS#L7

**Review by Tratcher — APPROVED** [review]

File: xml/Microsoft.Extensions.Caching.StackExchangeRedis/RedisCacheOptions.xml [hunk 1]
```diff
@@ -134,7 +134,7 @@
         <ReturnType>Microsoft.Extensions.Caching.StackExchangeRedis.RedisCacheOptions</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets the default configured <typeparamref name="TOptions" /> instance.</summary>
         <value>To be added.</value>
```
Comment (carlossanlop) [inline xml/Microsoft.Extensions.Caching.StackExchangeRedis/RedisCacheOptions.xml:138]:
```suggestion
        <value>An object that represents the Redis cache options.</value>
```

File: xml/Microsoft.Extensions.Caching.SqlServer/SqlServerCacheOptions.xml [hunk 2]
```diff
@@ -157,7 +157,7 @@
         <ReturnType>Microsoft.Extensions.Caching.SqlServer.SqlServerCacheOptions</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets the default configured <typeparamref name="TOptions" /> instance.</summary>
         <value>To be added.</value>
```
Comment (carlossanlop) [inline xml/Microsoft.Extensions.Caching.SqlServer/SqlServerCacheOptions.xml:161]:
```suggestion
        <value>An object that represents the SQL Server cache options.</value>
```

File: xml/Microsoft.Extensions.Caching.Redis/RedisCacheOptions.xml [hunk 3]
```diff
@@ -142,7 +142,7 @@
         <ReturnType>Microsoft.Extensions.Caching.Redis.RedisCacheOptions</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets the default configured <typeparamref name="TOptions" /> instance.</summary>
         <value>To be added.</value>
```
Comment (carlossanlop) [inline xml/Microsoft.Extensions.Caching.Redis/RedisCacheOptions.xml:146]:
```suggestion
        <value>An object that represents the Redis cache options.</value>
```

**Review by maryamariyan — APPROVED** [review]

**Review by carlossanlop — COMMENTED** [review]
@TianqiZhang @joelmartinez there are a couple of [weird warnings in the CI](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C10%5C14%5C500c749d-de78-3d4a-00e4-269d599bf9ec%5CPullRequest%5C202010140632335678-4923%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Rjao%2FjV9fgZWiLQkcbkXp4J0Hzq2ZHitUs5oZSBDuFQ%3D&st=2020-10-14T06%3A46%3A57Z&se=2020-11-14T06%3A51%3A57Z&sp=r), coming from APIs that I did not modify. The DocIds are not being recognized:

## StackExchange.Redis.ConfigurationOptions
https://github.com/dotnet/dotnet-api-docs/blob/6a4a5579c7e5e9161443959ff4b33a2b0b0d9199/xml/Microsoft.Extensions.Caching.Redis/RedisCacheOptions.xml#L75-L89

## Microsoft.Extensions.Internal.ISystemClock
https://github.com/dotnet/dotnet-api-docs/blob/6a4a5579c7e5e9161443959ff4b33a2b0b0d9199/xml/Microsoft.Extensions.Caching.SqlServer/SqlServerCacheOptions.xml#L193-L212

**Pull request merged**